Dockerfile.assets

```diff
@@ -58,3 +23,12 @@ COPY --chown=elife:elife source/ ${PROJECT_FOLDER}/source
 
 COPY --chown=elife:elife assets/sass ${PROJECT_FOLDER}/assets/sass
 RUN gulp --environment ${environment} generateStyles
+
+FROM busybox:1.28.3
```

Comment:
If this is going to be purely files without executables, you could try [`FROM scratch`](https://hub.docker.com/_/scratch/)

Reply:
Although `docker-compose.yml` (being oriented to services) needs a command to execute so easier to have `sh` around

Reply:
Yeah, tried that first. Wasn't sure what to use as `command`.